mempool/src/lib.rs

```diff
@@ -56,6 +61,8 @@ impl<'env> Mempool<'env> {
                 transactions_by_sender: HashMap::new(),
                 transactions_by_recipient: HashMap::new(),
                 transactions_sorted_fee: BTreeSet::new(),
+                filter: filter::MempoolFilter::new(filter::MempoolFilter::DEFAULT_RULES,
```

Comment:
Implement `Default` for `MempoolFilter` and change to:
```suggestion
                filter: MempoolFilter::default(),
```